monitor.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
```

Comment:
remove this file. It should be in your own utility collection if you find this script useful.